partiql-planner/src/main/kotlin/org/partiql/planner/internal/util/BinderUtils.kt

```diff
@@ -0,0 +1,66 @@
+package org.partiql.planner.internal.util
+
+import org.partiql.ast.Identifier
+import org.partiql.ast.Identifier.Simple
+import org.partiql.ast.Literal
+import org.partiql.ast.expr.Expr
+import org.partiql.ast.expr.ExprCast
+import org.partiql.ast.expr.ExprLit
+import org.partiql.ast.expr.ExprPath
+import org.partiql.ast.expr.ExprSessionAttribute
+import org.partiql.ast.expr.ExprVarRef
+import org.partiql.ast.expr.PathStep
+
+internal object BinderUtils {
```

Comment:
(self-review) previously, these were all top-level internal functions. moved to be under a designated `internal object`